### PR TITLE
cli: bulk reindexing

### PR DESCRIPTION
**Pull request by kaplun**
Signed-off-by: Samuele Kaplun <samuele.kaplun@cern.ch>

### CODE REVIEW

File: invenio_workflows_ui/cli.py [hunk 1]
```diff
@@ -56,8 +59,26 @@ def reindex(yes_i_know, data_type):
     query = WorkflowObjectModel.query.filter(
         WorkflowObjectModel.data_type.in_(data_type)
     )
+    indexer = workflow_api_class.indexer
+    req_timeout = current_app.config.get('INDEXER_BULK_REQUEST_TIMEOUT', )
```
Comment (david-caro) [inline invenio_workflows_ui/cli.py]: current_app is undefined no?

**Comment by david-caro** [comment]
Just peped 8 it and added the missing import... let's see the tests

File: invenio_workflows_ui/cli.py [hunk 2]
```diff
@@ -56,8 +58,53 @@ def reindex(yes_i_know, data_type):
...
+                yield {
+                    '_id': workflow_api_object.id,
+                    '_index': index,
+                    '_type': doc_type,
+                    'doc': body,
+                }
+    success, failures = elasticsearch.helpers.bulk(
+        es,
+        actions(),
+        stats_only=True,
```
Comment (kaplun) [inline invenio_workflows_ui/cli.py]: Actually I'd like to remove this flag and...

File: invenio_workflows_ui/cli.py [hunk 3]
```diff
@@ -56,8 +58,53 @@ def reindex(yes_i_know, data_type):
...
-            workflow_object = WorkflowObject.get(result.id)
-            workflow_api_class.create(workflow_object)
+    if failures:
+        click.secho(
+            "{} entries failed during reindexing while {} succeeded.".format(
+                failures,
+                success,
+            ),
+            fg='red',
+        )
```
Comment (kaplun) [inline invenio_workflows_ui/cli.py]: ... printing here the failures...

**Comment by david-caro** [comment]
okok, all yours 👍

**Comment by kaplun** [comment]
What about now?

**Comment by david-caro** [comment]
it's complaining about a small pep8 issue:
```
/home/travis/build/inveniosoftware-contrib/invenio-workflows-ui/invenio_workflows_ui/cli.py:77:21: E225 missing whitespace around operator

body=indexer._prepare_record(
```

**Pull request merged**